### ISSUE TITLE
[UX] Visualizer for Earmuff Mode

### ISSUE THREAD

**Issue by Zyro1331** [Yellow-Dog-Man/Resonite-Issues#3453]
### Is your feature request related to a problem? Please describe.

Adjusting the settings for Earmuff mode can be a little difficult to do without trial and error, finding the settings you find comfortable can take a bit of time and many other users to be actively speaking to understand the range of the settings you are adjusting.

### Describe the solution you'd like

A way to visualize the Earmuff Mode range. Much like the way the Whisper Bubble has an PBS_IntersectSpecular material to visualize the range of users who can hear you, the Earmuff Mode should have a function to let you see what range is required for you to hear other users. 
This would allow it to be easier for anyone to adjust the various Earmuff settings while also being able to visually see what's changing in real-time. 
You could also leave the option enabled to know the distance required at all times, so you can know how close you have to be to another user before going out-of-range.

### Describe alternatives you've considered

Making an Earmuff Mode visualizer from scratch, that uses the same formula as Resonite's built-in Earmuff Mode.

### Additional Context

I've overheard the idea of this feature multiple times from a few different people, and I'm surprised no one has directly asked for it as a Feature Request yet.
VRC has a function much like this by default for it's Earmuff Mode, which shows the projection of a sphere on the ground, and shows the distance required before people enter hearing range.

### Requesters

@zyro1331

**Comment by Hayden-Fluff** [comment]
I've made a proof of concept for this, and while it *kind of* works, it doesn't seem to actually correspond to what I'm hearing in game? At least with the Angle option.

https://storage.googleapis.com/haydens-video-storage/_temp/2024-12-31%2008-05-50.mp4
*GitHub wouldn't let me upload it, so you get an external link instead*

I've also exported this, it's a "facet" so you'd just have to save it to the inventory then spawn in userspace to get it to work.
*As a package since I am unable to save to my cloud*
https://storage.googleapis.com/haydens-video-storage/_temp/EarMuff%20Visualization.resonitepackage
